pkgs/development/tools/rust/rust-analyzer/ignore-git-and-rustfmt-tests.patch

```diff
@@ -7,12 +7,4 @@
 +#[ignore]
  fn check_code_formatting() {
      let _dir = pushd(sourcegen::project_root()).unwrap();
-     let _e = pushenv("RUSTUP_TOOLCHAIN", "stable");
-@@ -138,6 +139,7 @@ fn check_cargo_toml(path: &Path, text: String) -> () {
- }
- 
- #[test]
-+#[ignore]
```

Comment:
Since git tests are removed upstream, we could rename this patch file to `ignore-rustfmt-tests.patch`.

Reply:
Good point, will do!